lib/Extractor/Solver.cpp

```diff
@@ -103,7 +104,7 @@ class BaseSolver : public Solver {
         if (EC)
           return EC;
         if (!IsSat) {
-          RHS = I;
+          RHSs.emplace_back(I);
```

Comment:
do you want to return here, or try to find more RHSs?

Reply:
These are both constant synthesis, why we wants to collect multiple RHSs if a constant fit the LHS

Reply:
well, it depends on why we want multiple RHSs. for manasij's purposes a constant is always good, but for building a cost model I'd like as many alternatives as possible

Reply:
ping me if you need this.